django/db/migrations/serializer.py

```diff
@@ -269,6 +269,20 @@ def serialize(self):
         return "uuid.%s" % repr(self.value), {"import uuid"}
 
 
+class UnserializablesSerializer(BaseSerializer):
```

Comment:
"UnserializablesSerializer" seems like a poor name considering the thing can be serialized. Is it more like "class serializer"?